packages/big-design/src/components/Radio/Radio.tsx

```diff
@@ -52,6 +54,14 @@ const RawRadio: React.FC<RadioProps & PrivateProps> = ({
     warning('label must be either a string or a RadioLabel component.');
   }, [disabled, id, label, labelId]);
 
+  const renderedDescription = useMemo(() => {
+    if (!description || disabled) {
```

Comment:
Why don't we want to render the description if it's disabled?

Reply:
we don't have styles yet. Disabled render the main text as secondary40, which probably means the description should be something around secondary30? that means almost unreadable? @valterfatia

Reply:
I was about to say lets do secondary40 on both, but actually the text of the labels isn't meant to change based on the state of the input as per the design spec on https://www.figma.com/file/FnGsTa6UoCWQaQoOA7HobiDb/Form-Fields?node-id=195%3A26 
![image](https://user-images.githubusercontent.com/25913901/76267976-9c13af00-62c0-11ea-8297-d5b3874cdc8d.png)